### PR TITLE
Add details of vulnerability disclosure program

### PR DESCRIPTION
**Pull request by pritchyspritch**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
 
 ## Reporting a Vulnerability
 
-GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it. Quick links can be found at our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt).
```
Comment (gabrielcurrie) [inline SECURITY.md]: Suggest being a bit clearer on what we mean by "quick links", for example, by saying something like...

`This notice provides details for how you can let us know about vulnerabilities, or alternatively you can view our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt) file which contains quick links to contact us.`

File: SECURITY.md [hunk 2]
```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
...
 
-Send details to [disclosure@digital.cabinet-office.gov.uk], including:
+You can [report a vulnerability to GDS using HackerOne](https://hackerone.com/44c348eb-e030-4273-b445-d4a2f6f83ba8/embedded_submissions/new), please include:
 - the website, page or repository where the vulnerability can be observed
 - a brief description of the vulnerability 
+- details of the steps we need to take to reproduce the vulnerability
 - optionally the type of vulnerability and any related [OWASP category]
 - non-destructive exploitation details
+- screenshots or logs if you have them
```
Comment (gabrielcurrie) [inline SECURITY.md]: Suggested wording tweak:

`screenshots or logs showing the exploitation of the vulnerability`

Comment (gabrielcurrie) [inline SECURITY.md]: Could we move the optional items to another list after this, headed with something like `If you are able to, please also include:`

File: SECURITY.md [hunk 3]
```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
...
 
-GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it. Quick links can be found at our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt).
 
-Send details to [disclosure@digital.cabinet-office.gov.uk], including:
+You can [report a vulnerability to GDS using HackerOne](https://hackerone.com/44c348eb-e030-4273-b445-d4a2f6f83ba8/embedded_submissions/new), please include:
 - the website, page or repository where the vulnerability can be observed
 - a brief description of the vulnerability 
+- details of the steps we need to take to reproduce the vulnerability
 - optionally the type of vulnerability and any related [OWASP category]
```
Comment (gabrielcurrie) [inline SECURITY.md]: Suggested wording tweak:

`the type of vulnerability, for example, the [OWASP category](https://owasp.org/www-community/vulnerabilities/)`

Also suggest we move "any related" into a separate line?

File: SECURITY.md [hunk 4]
```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
 
 ## Reporting a Vulnerability
 
-GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it. Quick links can be found at our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt).
 
-Send details to [disclosure@digital.cabinet-office.gov.uk], including:
+You can [report a vulnerability to GDS using HackerOne](https://hackerone.com/44c348eb-e030-4273-b445-d4a2f6f83ba8/embedded_submissions/new), please include:
```
Comment (gabrielcurrie) [inline SECURITY.md]: Suggested wording change:

```
You can report a vulnerability to GDS through our vulnerability disclosure programme at [HackerOne](https://hackerone.com/44c348eb-e030-4273-b445-d4a2f6f83ba8/embedded_submissions/new). Alternatively, you can send an email to [disclosure@digital.cabinet-office.gov.uk]; if you do this you may get a response from ZenDesk, which is our ticketing system. 

When reporting a vulnerability to us, please include:
```

File: SECURITY.md [hunk 5]
```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
...
-- reports indicating that our services do not fully align with “best practice”, for example missing security headers
+- break the law
+- access unnecessary or excessive amounts of data
+- modify data
+- use high-intensity invasive or destructive scanning tools to find vulnerabilities
+- try a denial of service - for example overwhelming a service on GOV.UK with a high volume of requests
+- disrupt GOV.UK’s services or systems
+- tell other people about the vulnerability you have found until we have disclosed it
+- social engineer, phish or physically attack our staff or infrastructure
+- demand money to disclose a vulnerability
```
Comment (gabrielcurrie) [inline SECURITY.md:40]: These are less things that fall in/out of scope, and more the things that we don't want people to do to find a vulnerability, or do when they find a vulnerability. Agree they should be in here, but can we tweak wording to be clearer?

File: SECURITY.md [hunk 6]
```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
...
 
-If you are not sure, you can still reach out via email; [disclosure@digital.cabinet-office.gov.uk].
+Only submit reports about exploitable vulnerabilities through HackerOne.
+
+If you are not sure, or you have found:
+- a non-exploitable vulnerability
+- something you think could be improved - for example, missing security headers
+- TLS configuration weaknesses - for example weak cipher suite support or the presence of TLS1.0 support
+
+Then you can still [reach out via email](disclosure@digital.cabinet-office.gov.uk) or using the [GOV.UK contact form](https://www.gov.uk/contact/govuk).
```
Comment (gabrielcurrie) [inline SECURITY.md]: It feels like this should go into the top section on Reporting a Vulnerability, rather than this section on scope?

**Review by gabrielcurrie — CHANGES_REQUESTED** [review]
Comments in line, sorry I couldn't get the "suggest changes" function to work!

**Review by gabrielcurrie — APPROVED** [review]
Looks good!

**Review by akinnane — APPROVED** [review]

**Pull request merged**